docs/tutorials/getting-started.md

```diff
@@ -147,7 +147,7 @@ the `<py-script>` tag to write to.
 
 Use the `<py-config>` tag to set and configure general metadata along with declaring dependencies for your PyScript application. The configuration has to be set in either TOML or JSON format. If you are unfamiliar with JSON, consider reading [freecodecamp's JSON for beginners](https://www.freecodecamp.org/news/what-is-json-a-json-file-example/) guide for more information. And for TOML, consider reading about it [here](https://learnxinyminutes.com/docs/toml/).
 
-The ideal place to use `<py-config>` in between the `<head>...</head>` tags.
+The ideal place to use `<py-config>` in between the `<body>...</body>` tags.
```

Comment:
this sentence sounds weird to me. It's not "ideal", it's the only place where it is allowed.
I would remove it altogether or tell people that the "conventional" place where to put it is just after the opening of `<body>`.